pgbot/utils.py

```diff
@@ -388,3 +388,16 @@ def code_block(string: str, max_characters=2048):
         return f"```\n{string[:max_characters - 7]} ...```"
     else:
         return f"```\n{string[:max_characters]}```"
+
+
+def remove_all(list_thing, thing_to_remove):
```

Comment:
Why is this here? These functions are not used anywhere who should they be removed?